### DISCUSSION TITLE
Option for deactivating creating new rooms

### DISCUSSION THREAD

**Discussion by rinjuu123** [dyc3/opentogethertube#1110] [Q&A] [answered]
Hello, is there a way to temporaly deactivate "create new room"?

**Comment by dyc3** [comment]
I'm assuming you are talking about self hosting? Currently no, there isn't.

As a workaround, in your reverse proxy, you can block requests to `/api/room/create` to block creating permanent rooms, and `/api/room/generate` to block creating temporary rooms.

**Reply by rinjuu123** [reply]
Yes that's i mean. I will try that. Thank you :)
Do you consider to make it a feature?

**Reply by dyc3** [reply]
Sure, I'll add some config options for it.

**Reply by dyc3** [reply, marked as answer]
Added `room.enable_create_temporary` and `room.enable_create_permanent` in #1111

**Reply by rinjuu123** [reply]
I don't know if you already know this, but it fails "Rust / build (pull_request)" Check on the auto merge.
